content/r-package-dev/Version_Control.md

```diff
@@ -3,7 +3,7 @@ author: Lindsay R. Carr
 date: 9999-10-31
 slug: git
 title: Version Control
-draft: True
+draft: FALSE 
```

Comment:
I've only ever removed draft: true, not changed it to false. I assume this will work, but not 100% sure